README.md

```diff
@@ -37,6 +37,63 @@ go get github.com/aws-controllers-k8s/dev-tools/cmd/ackdev
 
 Call `ackdev help` for detailed usage instructions.
 
+### Setup
+
+To be able to use `ackdev` you'll have to run `ackdev setup` before any other command.
+
+For example
+```bash
+ackdev setup --root-directory $WORKDIR --services s3,ecr,sqs,sns
+```
+
+The setup command will simply create a yaml file named `$HOME/.ackdev.yaml`
+(you can choose a different file path `--config-file`)
+
+**NOTE**: If you are already a contributor you probably want to use `$GOPATH/src/github.com/aws-controllers-k8s` as `$WORKDIR`. 
+If you only want to test the tool, 
+We recommended you to set your `$WORKDIR` to a different directory.
+
+> Will my projects work/compile if i use a different directory than `$GOPATH/src/github.com/aws-controllers-k8s` ?
+
+Yes, even if it's not inside `GOPATH`, it will work as expected.
+Since Go 1.11.4 the Go compiler doesn't rely **entirely** on GOPATH to build projects.
+
+
+The generated configuration file will look like:
+
+``` yaml
+rootDirectory: /home/amine/go/source/github.com/aws-controllers-k8s/dev-tools
+git:
+  sshKeyPath: ""
+github:
+  token: ""
+  username: ""
+  forkPrefix: ""
+repositories:
+  core:
+  - runtime
+  - dev-tools
+  - community
+  - code-generator
+  services:
+  - s3
+  - ecr
+  - sns
+  - sqs
+run:
+  flags: {}
+```
+
+To use all `ackdev` features you will need to fill the `git.sshKeyPath` and `github` sections.
```

Comment:
I use `git` with HTTPS, not ssh. (I know it's unsafe, I just hate messing with key files). What are my alternatives?

Reply:
> I use `git` with HTTPS, not ssh. (I know it's unsafe, I just hate messing with key files). What are my alternatives?

I don't think we should support an HTTPS alternative, @RedbackThomson, at least not for any repositories that we want to push to.

Reply:
Just for the sake of security? I just believe that 99% of people use HTTPS, so I'm sure this will be a recurring problem